src/openfermion/third_party/representability/_bijections.py

```diff
@@ -0,0 +1,63 @@
+from typing import Callable, List, Tuple
+
+
+class Bijection(object):
+
+    def __init__(self, fwd: Callable, rev: Callable, sizes: Callable):
+        """
+        Bijection holds forward maps and backwards maps
+
+        Args:
+            fwd: forward mapping function
+            rev: backwards mapping function
+            sizes: function getting tuple of domain and codomain element
+                   dimensions
+        """
+        self.fwd = fwd
+        self.rev = rev
+        self.domain_element_sizes = sizes
+
+
+def index_index_basis(n: int) -> Bijection:
+    """
+    Create an index-to-index Bijection
+
+    Args:
+        n: length of basis
+    Returns: A Bijection object
+    """
+    idx_dict = dict(zip(range(n), range(n)))
+
+    def forward(i: int) -> int:
```

Comment:
If I define it as variable then it's not callable? I want this to be a function.

Reply:
I think @vtomole meant `forward = idx_dict.get`

Reply:
But then you have to remember that .get is a function built into dict.  I think this is much clearer since all instantiations of `Bijection` will need to define these functions.